charts/newrelic-infrastructure/values.yaml

```diff
@@ -68,8 +68,32 @@ serviceAccount:
   # Specify any annotations to add to the ServiceAccount
   annotations:
 
-# If you wish to provide additional labels to apply to the pod(s), specify
-# them here
+# For unprivileged mode.  Default is 1000.
+# Can also be modified for OpenShift clusters.
+# https://www.openshift.com/blog/a-guide-to-openshift-and-uids
+runAsUser:
+
+### Openshift 4.x ###
+# Set openshift: true if installing in an OpenShift 4.x Cluster
+openshift: false
+
+# Openshift 4.x control plane endpoints
+# https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring#openshift-4x-configuration
+# schedulerEndpointUrl: "https://localhost:10259"
+# etcdEndpointUrl: "https://localhost:9979"
+# controllerManagerEndpointUrl: "https://localhost:10257"
+# apiServerEndpointUrl: "https://localhost:6443"
+
+# ETCD mTLS
+# https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring#mtls-how-to
+# https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring#mtls-how-to-openshift
```

Comment:
same here, I would explain a bit why adding these links.

It might help to put something in the readme as well :)